docs/docs/general/03-Element Selection/recommended-practices.md

```diff
@@ -0,0 +1,171 @@
+---
+sidebar_position: 2
+title: Recommended Practices
+---
+
+# Recommended Practices
+On this page you will learn the different methods to detect elements. Every method has its strength and weaknesses, for example one is more accurate than another but also more effort when using it.
+
+## The Overarching Goal
+As a user you want to detect elements reliably. There are two axes of reliability:
+
+* False Positives: Detecting the wrong element or detecting more elements than you want to detect.
+* False Negatives: Detecting no element where there is one.
```

Comment:
Different criteria instead of overarching goal

- Sensitivity
- Specificity
- Stability/Reliability/Flakiness (how stable is the detection over runs)
- Speed of execution
- Can I describe it? / Use case
- Ease of writing
- Effort of maintaining it
- etc.

with very specific description/definition, e.g., with images, tables (e.g., confusion matrix)